### ISSUE TITLE
0 confirms, not indicated on blockchain

### ISSUE THREAD

**Issue by Attempt12** [darkwallet/darkwallet#173]
Hey,

How to proceed with my current problem, which is that I have coins that were supposed to have sent from Darkwallet, but for hours it is remaining at 0 confirmations, and the transaction is not shown on Blockchain. The account balance of my account is the same, but if I try to resend, it says that I have 0 coins available. I tried restarting the browser to no avail. How do I make these coins arrive and be seen by blockchain?

Thanks

**Comment by dabura667** [comment]
What kind of transaction was it?
Sent from a multisig?
Sent to a stealth?
Did you check other block explorers? blockr.io? live.insight.is?

**Comment by Attempt12** [comment]
It was an ordinary transaction that went from savings pocket to a known wallet address that I control. It seems perhaps it was not broadcasted. The transaction is not visible on any block explorers, and remains at 0 confirms.

**Comment by Attempt12** [comment]
Update: Originally I attempted to resend the coins, but it failed saying i Had 0 Coins. On another attempt I was able to send out some coins (despite the unfulfilled transaction), but with another attempt to finish sending the remaining coins, I got the same error, 0 coins available. After attempting to send the coins from the pocket UI for sending, as opposed to the sending tab, it was successful. This has been mildly frustrating, but it seems the coins are all going now!!

**Comment by amingilani** [comment]
I can confirm. Keeps happening to me too. Coinmixing was stuck for three days, showing an unconfirmed balance in negative. Finally, today it showed up as a positive balance and I sent it by manually disabling Coinjoin.

**Comment by InnovateLiberate** [comment]
Any update on how to resolve this issue? I sent funds from my spending account to the stealth address of my business account and I am currently stuck at 0% for almost 24 hours. I attempted to send a couple bits a couple hours after the original transaction to an outside wallet to see if the system recognized the failed broadcast and recredit my coins, I was able to set up a transaction but now its stuck at 0% conformations too right next to the original transactions to my stealth address.

**Comment by millsdmb** [comment]
If I might suggest to export the private key from dark wallet and import to
blockchain.info

**Comment by millsdmb** [comment]
However, I am experiencing the same problem as everyone else. Almost looks like an obelisk issue.

![image](https://cloud.githubusercontent.com/assets/7471116/5308242/ff88db52-7be2-11e4-8ebf-2df9805e39bb.png)

this was confirmed days ago.

**Comment by InnovateLiberate** [comment]
yeah that's what mine looks like.... :/

**Comment by VivaLaPanda** [comment]
Same for me here... Anyone tried with different Obelisk servers?

**Comment by caedesvvv** [comment]
Restarted unsystem obelisk server to see if it helps.

**Comment by VivaLaPanda** [comment]
Nope, still stuck at 0 confirmations, and not appearing on blockchain.info.

**Comment by VivaLaPanda** [comment]
Could it be a NAT issue? My ISP doesn't leave many ports open.

**Comment by caedesvvv** [comment]
It can't be a nat issue. If the tx shows as no confirmations in darkwallet and doesn't show in blockchain.info or other software it means it didn't get sent.

In 0.7 or before you can clear tasks, restart the browser so it clears reserved funds and try again.

**Comment by VivaLaPanda** [comment]
How can I clear tasks? It's not too big of a deal, but restarting the browser doesn't fix it. Just restarting the extension will allow you to spend the money that was sent, but it doesn't matter since the transaction still won't go through.

**Comment by VivaLaPanda** [comment]
Ok, I confirmed the problem. It is an issue with CoinJoin. First: To those of you having problems. Clear tasks, then reset the extension. This will get rid of all the broken transactions. Then do whatever transactions via send UI with CoinJoin disabled. It appears that after a CoinJoin is completed, the transaction isn't getting broadcasted. (Running 0.7)

**Comment by millsdmb** [comment]
tried clearing tasks, no effect. its a pocket to pocket tx, one says sending, the other says receiving. tx not on blockchain, but exporting privkey seemed to work. I can import it to blockchain.info to recover funds that way -- unsure of the root cause in darkwallet however. Wish I could help, but no clue what's happening.

**Comment by VivaLaPanda** [comment]
Did you got into chrome settings and reset after clearing tasks?

**Comment by millsdmb** [comment]
yes

**Comment by InnovateLiberate** [comment]
Ok so after tinkering around with the wallet for a bit I finally have confirmation on the blockchain! I am a hardware guy, not a software person, so I cannot tell you what exactly I did right but I will tell you everything I did and every button I pushed. 
-SettingsTab-Enforced mixing: off
-Pocket/Spending/ActionTab-Mix funds on this pocket (CoinJoin): off
-SettingsTab/Tools-Enable Developer mode: on
-SettingsTab/Tools: Clear Notifications: yes
-SettingsTab/Tools-Export Private Keys: export all (I did not enter any address)

After that I returned to the main hub with all my pockets and attempted to transfer some coin again through-MyPockets/Spending/Overview/Send (used a regular address from another service)

Success! Hash appears in both the block explorer function of DW and that of blockchain.info
Hope this helps
